src/screens/Asset/index.js

```diff
@@ -142,6 +143,9 @@ class Asset extends PureComponent<AssetProps, any> {
           counterValueCurrency={counterValueCurrency}
           renderTitle={this.renderListHeaderTitle}
         />
+        <View style={[styles.fabActions]}>
+          <FabActions />
+        </View>
```

Comment:
Do you need the actions to be sticky ?

Reply:
Not too sure what you mean 😅  Could you explain, and also perhaps @IAmMorrow knows?